src/modules/commandrunner/src/lib/CommandRunner.cpp

```diff
@@ -412,7 +420,7 @@ void CommandRunner::WorkerThread(CommandRunner& instance)
         instance.m_commandQueue.Pop();
     }
 
-    OsConfigLogInfo(CommandRunnerLog::Get(), "Worker thread stopped for %s", instance.m_clientName.c_str());
+    OsConfigLogInfo(CommandRunnerLog::Get(), "Worker thread stopped for '%s'", instance.m_clientName.c_str());

```

Comment:
Since each `CommandRunner` instance is paired with a unique `clientName`, every session opened with the same `clientName` will point to the same instance. A separate *session id* (generated at each successful `MmiOpen()`) could be used to distinguish between the different sessions and used in log messages. What do you think?

Reply:
Sounds good.